Vostok.Hosting.Abstractions/PublicAPI.Shipped.txt

```diff
@@ -179,4 +179,5 @@ Vostok.Hosting.Abstractions.Requirements.RequiresPort
 Vostok.Hosting.Abstractions.Requirements.RequiresPort.RequiresPort() -> void
 Vostok.Hosting.Abstractions.Requirements.RequiresSecretConfiguration
 Vostok.Hosting.Abstractions.Requirements.RequiresSecretConfiguration.RequiresSecretConfiguration(System.Type type, params string[] scope) -> void
-Vostok.Hosting.Abstractions.WellKnownApplicationIdentityProperties
\ No newline at end of file
+Vostok.Hosting.Abstractions.WellKnownApplicationIdentityProperties
```

Comment:
New changes should be placed in Unshipped file, then `vostok-release` tool will rotate it